run.ps1

```diff
@@ -127,6 +127,6 @@ Add-Content $Env:GITHUB_PATH "$pwd\php-sdk\msys2\usr\bin"
 Add-Content $Env:GITHUB_PATH "$pwd\php-bin"
 Add-Content $Env:GITHUB_PATH "$pwd\php-dev"
 
-Write-Output "::set-output name=toolset::$toolset"
-Write-Output "::set-output name=prefix::$pwd\php-bin"
-Write-Output "::set-output name=vs::$vs"
+Write-Output "toolset=$toolset" >> $GITHUB_OUTPUT
+Write-Output "prefix=$pwd\php-bin" >> $GITHUB_OUTPUT
+Write-Output "vs=$vs" >> $GITHUB_OUTPUT
```

Comment:
An alternative syntax can be found in https://github.com/orion-global/actions/blob/13739401a9d41b1b49a9ff6d49dcda2993abf3e1/snyk_results/action.yml#L53:

```
Write-Output "VAR=$VAR" | Out-File -FilePath $ENV:GITHUB_OUTPUT -Encoding utf8 -Append
```

I see you used `$Env:GITHUB_PATH` earlier to refer to an environment variable, so you might prefer to use this syntax instead of what I suggested here. I'm not sure how to test this locally so I'll defer to you on that decision.

And no worries if you end up making the change yourself and closing this PR w/o merging :v:

Reply:
Thank you for the PR!

> I see you used `$Env:GITHUB_PATH` earlier to refer to an environment variable,

I did some experiments, but failed to solve the issue. Did you test your changes? If so, I'll gladly merge this PR; otherwise, I'll do some tests within the next days. And no, I don't particularly mind the exact syntax.

Reply:
I originally had no idea how to test this, but it looks like I can do so by modifying the `uses` line in our workflow to point to my fork and branch (per: [Using release management for your custom actions](https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions#using-release-management-for-your-custom-actions)). I'll get to work on that and follow up once I have a passing build.